pages/glossary/owner-element.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Owned by
+key: owned-by
+---
+
+An element A is owned by element B if:
+
+1. Both elements exist within the same [document tree](https://www.w3.org/TR/dom41/#document-trees) or [shadow tree](https://www.w3.org/TR/dom41/#shadow-trees), and element A is the first element within the tree, in [tree order](https://www.w3.org/TR/dom/#concept-tree-order), whose ID is included within the `aria-owns` attribute of element B, or
+2. Element B is the closest [shadow including ancestor](https://www.w3.org/TR/dom41/#concept-shadow-including-ancestor) of element A that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+Nodes that are not included in the accessibility tree cannot be owned by other elements.
+
+> **Note:** When using `aria-owns`, hiding the owner element through WAI-ARIA (`aria-hidden="true"`) or CSS (`display:none` or `visibility:hidden`) will not change the ownership, but should instead hide all elements owned by the element.
```

Comment:
This contradicts your point 1 and 2. I also don't think this is intuitive, and it doesn't seem to be how it is currently implemented. I would guess this is unlikely to ever become standardised.

Reply:
@WilcoFiers,

This part of your comment: 
> I also don't think this is intuitive, and it doesn't seem to be how it is currently implemented. I would guess this is unlikely to ever become standardised. 

... is that for the note or for the entire definition of "owned by"?

Reply:
"Owned by" definition has been slightly rewritten and note removed. 

Can I have you review again and say if you agree?